sklearn/metrics/_plot/calibration_curve.py

```diff
@@ -0,0 +1,193 @@
+
+        self.axs_ = ax
+        self.figure_ = ax.figure
+        return self
+
+
+@_deprecate_positional_args
+def plot_calibration_curve(estimator, X, y, *,
+                           normalize=False, n_bins=5, strategy='uniform',
+                           name=None, ref_line=True, ax=None, **kwargs):
+    """Plot calibration curve for binary classifiers.
+
+    Extra keyword arguments will be passed to matplotlib's `plot`.
+
+    Read more in the :ref:`User Guide <calibration>`.
+
+    Parameters
+    ----------
+    estimator : estimator instance
+        Fitted classifier or a fitted :class:`~sklearn.pipeline.Pipeline`
+        in which the last estimator is a classifier. The classifier must
+        have a `predict_proba` method; set `probability=True` for
+        :class:`~sklearn.svm.SVC` and :class:`~sklearn.svm.NuSVC`
+        (see: :ref:`User Guide <scores_probabilities>`) or use
+        :class:`~sklearn.calibration.CalibratedClassifierCV`.
+
+    X : {array-like, sparse matrix} of shape (n_samples, n_features)
+        Input values.
+
+    y : array-like of shape (n_samples,)
+        Binary target values.
+
+    normalize : bool, default=False
+        Whether the output of `estimator.predict_proba(X)` needs to be
+        normalized into the [0, 1] interval (i.e. is not a proper probability).
+        If `True`, the smallest value is linearly mapped onto 0 and the largest
+        one onto 1.
+
+    n_bins : int, default=5
+        Number of bins to discretize the [0, 1] interval into when calculating
+        the calibration curve.
+
+    strategy : {‘uniform’, ‘quantile’}, default=’uniform’
+        Strategy used to define the widths of the bins.
+
+        uniform
+            The bins have identical widths.
+        quantile
+            The bins have the same number of samples and depend on `y_prob`.
+
+    name : str, default=None
+        Name for labeling curve. If `None`, the name of the estimator is used.
+
+    ref_line : bool, default=True
+        If `True`, plots a reference line representing a perfectly calibrated
+        classifier.
+
+    ax : matplotlib axes, default=None
+        Axes object to plot on. If `None`, a new figure and axes is created.
+
+    **kwargs : dict
+        Keyword arguments to be passed to :func:`matplotlib.pyplot.plot`.
+
+    Returns
+    -------
+    display : :class:`~sklearn.metrics.CalibrationDisplay`
+        Object that stores computed values.
+    """
+    check_matplotlib_support("plot_calibration_curve")
+
+    classification_error = ("{} should be a binary classifier".format(
+        estimator.__class__.__name__))
+    if not is_classifier(estimator):
+        raise ValueError(classification_error)
+
+    prediction_method = _check_classifer_response_method(
+        estimator, response_method='predict_proba'
+    )
+
+    y_prob = prediction_method(X)
+
+    if y_prob.ndim != 1:
+        if y_prob.shape[1] != 2:
+            raise ValueError(classification_error)
```

Comment:
Maybe happend "It seems that the input is multiclass" or something like that to the error message so that users know what to do.